ai/GameDomainObjects/field.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, balls: List[Ball]):
     @property
     def ball(self):
         if len(self._balls) == 0:
-            raise RuntimeError("There is no ball to detect")
+            return None
```

Comment:
La plupart des calls à ball check la position de la balle, donc au lieu d'avoir un runtime error claire, ils vont recevoir une erreur comme 'object None does not have an attribute position'

Reply:
ca faisait crashé pas mal de truc et c'etait merdeux à catché